tests/out_of_proc_quote_generation/out_of_proc_quote_generation.edl

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/edl/attestation.edl" import *;
+    from "openenclave/edl/logging.edl" import oe_write_ocall;
+    from "openenclave/edl/syscall.edl" import *;
```

Comment:
you shouldn't need this

Reply:
A quick test revealed that I still need `syscall.edl`, but not the other two.